### ISSUE TITLE
Fix existing react-a11y warnings

### ISSUE THREAD

**Issue by toolness** [mozilla/teach.webmaker.org#138]
As of the merging of #137, this is what we need to do:

- [x] Fix all occurrences of `You have a click handler on a non-interactive element but no 'role' DOM property. It will be unclear what this element is supposed to do to a screen-reader user. http://www.w3.org/TR/wai-aria/roles#role_definitions`
- [x] Fix all occurrences of `You have a click handler on a non-interactive element but no 'tabIndex' DOM property. The element will not be navigable or interactive by keyboard users. http://www.w3.org/TR/wai-aria-practices/#focus_tabindex`
- [x] Fix all occurrences of `You have a click handler on an element with no screen-readable text. Add 'aria-label' or 'aria-labelled-by' attribute, or put some text in the element.`
- [x] Once the above are done, get rid of the `ENABLE_REACT_A11Y` environment variable check in `gulpfile.js` so that **accessibility warnings always cause the test suite to fail**.

**Comment by toolness** [comment]
@gesa, are you interested in trying this one?

**Comment by gesa** [comment]
Ahhhh! Yes, I am interested! I **can** do this! (i think)

**Comment by alicoding** [comment]
@gesa have you started working on this one? If not can you assign this to me? otherwise I will try look for something else :P
